ideas/313-core-networking.md

```diff
@@ -0,0 +1,67 @@
+---
+id: #313-core-networking
+title: Core Networking Swarm Proposal
+status: research/specification/implementation/maintenance
```

Comment:
status research

Also this is about _network incentives_ right? Not just network

Please add me as additional contributor

Reply:
Yeah, I just choose it as a more generic term, but sure, I can add `incentives` too.